study-builder/studies/testboston/study-events.conf

```diff
@@ -479,5 +479,24 @@
       "dispatchToHousekeeping": false,
       "order": 3
     }
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": ${id.act.result_report},
+        "statusType": "CREATED"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "templates": [
+          { "emailTemplate": ${emails.en_result_report_created}, "language": "en", "isDynamic": true },
+          { "emailTemplate": ${emails.es_result_report_created}, "language": "es", "isDynamic": true },
+          { "emailTemplate": ${emails.ht_result_report_created}, "language": "ht", "isDynamic": true }
+        ],
+        "pdfAttachments": []
+      },
+      "maxOccurrencesPerUser": 1,
```

Comment:
Remove this line. Otherwise, we'll only be able to send out email for one report.
```suggestion
```